app/grandchallenge/cases/serializers.py

```diff
@@ -171,6 +184,13 @@ def validate(self, attrs):
                 "Only one of archive, answer or reader study can be set"
             )
 
+        if "interface" in attrs and (
+            "reader_study" in attrs or "answer" in attrs
+        ):
```

Comment:
This would be more robust to changes:

```suggestion
        if "interface" in attrs and "archive" not in attrs:
```